client/src/components/App/Content/InvestigationForm/useInvestigationForm.ts

```diff
@@ -215,16 +216,54 @@ const useInvestigationForm = (): useInvestigationFormOutcome => {
         });
     };
 
+    const fetchStatuses = () => {
+        logger.info({
+            service: Service.CLIENT,
+            severity: Severity.LOW,
+            workflow: 'Fetching Statuses',
+            step: `launching sub statuses request`,
+            user: userId,
+            investigation: epidemiologyNumber
+        });
+        axios.get('/investigationInfo/statuses').then((result: any) => {
+            logger.info({
+                service: Service.CLIENT,
+                severity: Severity.LOW,
+                workflow: 'Fetching Statuses',
+                step: `recieved DB response ${JSON.stringify(result)}`,
+            });
+            const resultNodes = result?.data?.data?.allInvestigationStatuses?.nodes;
+
+            if (resultNodes) {
+                setStatuses(resultNodes.map((element: any) => element.displayName));
```

Comment:
maybe return only the names from ther server withouth displayName

Reply:
fixed